extensions/blocks/contact-info/contact-info.php

```diff
@@ -38,3 +38,34 @@ function jetpack_contact_info_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'contact-info' );
 	return $content;
 }
+
+add_filter( 'wp_kses_allowed_html', 'jetpack_contact_info_add_schema_attributes', 10, 2 );
+/**
+ * Add the schema attributes to post data. So that it can be saved as expected.
+ * @param array  $tags      Allowed tags by.
+ * @param string $context_type Context type (explicit).
+ */
+function jetpack_contact_info_add_schema_attributes( $tags, $context_type ) {
+	if ( $context_type !== 'post' ) {
+		return $tags;
+	}
+	l( 'jetpack_contact_info_add_schema_attributes' );
```

Comment:
We'll need to remember to remove this debug.

Reply:
Fixes